jobs/build/ocp/Jenkinsfile

```diff
@@ -755,44 +865,68 @@ Please direct any questsions to the Continuous Delivery team (#aos-cd-team on IR
     Jenkins job: ${env.BUILD_URL}
 
     BUILD / PUSH FAILURES:
     ${failed_map}
     """);
 
-                            def resp = input message: "Error during Image Build for OCP v${BUILD_VERSION}",
-                                    parameters: [
-                                            [$class     : 'hudson.model.ChoiceParameterDefinition',
-                                             choices    : "RETRY\nCONTINUE\nABORT",
-                                             description: 'Retry (try the operation again). Continue (fails are OK, continue pipeline). Abort (terminate the pipeline).',
-                                             name       : 'action']
+			    // Query the user on the CLI and get a response
+                            def resp = input(
+                                message: "Error during Image Build for OCP v${BUILD_VERSION}",
+                                parameters: [
+                                    [
+                                        name: 'action'
+                                        description: 'Retry (try the operation again). Continue (fails are OK, continue pipeline). Abort (terminate the pipeline).',
+                                        $class: 'hudson.model.ChoiceParameterDefinition',
+                                        choices: [
+                                            "RETRY",
+                                            "CONTINUE",
+                                            "ABORT"
+                                        ],join("\n")
                                     ]
-
-                            if (resp == "RETRY") {
-                                return false  // cause waitUntil to loop again
-                            } else if (resp == "CONTINUE") {
-                                echo "User chose to continue. Build failures are non-fatal."
-                                BUILD_EXCLUSIONS = failed_map.keySet().join(",") //will make email show PARTIAL
-                                BUILD_CONTINUED = true //simply setting flag to keep required work out of input flow
-                                return true // Terminate waitUntil
-                            } else { // ABORT
-                                error("User chose to abort pipeline because of image build failures")
+                                ]
+                            )
+
+                            // decide how to proceed after error
+                            switch (resp) {
+                                case "RETRY":
+                                    // cause waitUntil to loop again
+                                    return false
+                                case "CONTINUE":
+                                    echo "User chose continue. Build failures are non-fatal."
+                                    //will make email show PARTIAL
+                                    BUILD_EXCLUSIONS = failed_map.keySet().join(",")
```

Comment:
You added to this, but it was already set above. And won't get onto the `exclude` variable used below... I don't think anyways.